src/utils/oib.util.ts

```
@@ -0,0 +1,30 @@
+// This file is part of the `@domagojpa/oib-validation` package.
+export function isOibValid(input: string): boolean {
+    const oib = input.toString();
+
+    if (oib.match(/^\d{11}$/) === null) {
+        return false;
+    }
+
+    let calculated = 10;
+
+    for (const digit of oib.substring(0, 10)) {
+        calculated += parseInt(digit);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)